lib/paths.js

```diff
@@ -0,0 +1,17 @@
+// @ts-check
+const path = require('node:path');
+
+const REPOSITORY_ROOT_PATH = path.resolve(__dirname, '..');
+
+const APP_ROOT_PATH = path.resolve(__dirname, '..');
```

Comment:
These currently have the same value. When we move PrairieLearn code to `apps/prairielearn`, `APP_ROOT_PATH` will become the absolute path to the `apps/prairielearn` directory. I've kept this future usage in mind when picking between these constants in this PR - bear with the seemingly-arbitrary choices for now!